crates/bevy_ecs/examples/handling_errors_in_systems.rs

```diff
@@ -0,0 +1,52 @@
+use std::error::Error;
```

Comment:
```suggestion
//! Demonstrates different strategies that might be used to handle systems that could fail.

use std::error::Error;
```

Reply:
All of our examples use these module-docs to give a brief summary of what's being done.